2-mandatory/3-shopping-list.js

```diff
@@ -19,8 +19,25 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
-}
+  let shoppingList = {};
+  shoppingList.items =[];
+  shoppingList.name = recipe.name
+  // check ungredients
+  for (let ingredient of recipe.ingredients)
+  {
+    
+    if (pantry.fridgeContents.includes(ingredient)=== false 
```

Comment:
When you see something like this: `pantry.fridgeContents.includes(ingredient) === false` - you can re-write it like this: `!pantry.fridgeContents.includes(ingredient)`. Can you think of why that works?

Reply:
ok great thank you